### ISSUE TITLE
Superset not running. Error in config file

### ISSUE THREAD

**Issue by masterlittle** [apache/incubator-superset#3909]
Make sure these boxes are checked before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if any
- [x] I have reproduced the issue with at least the latest released version of superset
- [x] I have checked the issue tracker for the same issue and I haven't found one similar
When giving the SUPERSET_CONFIG_PATH as /home/superset/superset_config.py, I am getting the following error --
Traceback (most recent call last):
  File "/usr/local/bin/superset", line 12, in <module>
    from superset.cli import manager
  File "/usr/local/lib/python2.7/dist-packages/superset/__init__.py", line 21, in <module>
    from superset import utils, config  # noqa
  File "/usr/local/lib/python2.7/dist-packages/superset/config.py", line 350, in <module>
    override_conf = imp.load_source('superset_config', os.environ[CONFIG_PATH_ENV_VAR])
  File "/home/superset/superset_config.py", line 41, in <module>
    with open(PACKAGE_FILE) as package_file:
IOError: [Errno 2] No such file or directory: u'/home/superset/static/assets/package.json'

Looks like it is searching for the assets in a relative path to the config file. How to resolve this?
### Superset version
0.20.6

**Comment by amoussoubaruch** [comment]
@masterlittle Please Are you following step by step superset installation?

**Comment by masterlittle** [comment]
Yeah. It works if I don't give a config path. And now I got it working after hardcoding the BASE_DIR variable in my config file.
I had to change the value of BASE_DIR to '/usr/local/lib/python2.7/dist-packages/superset/'

I think the issue was that this line--
BASE_DIR = os.path.abspath(os.path.dirname(__file__))

It keeps the BASE_DIR as the same directory as the config file and does not build the static/assets path there.
